src/reducers/sponsors/badge-scans-list-reducer.js

```diff
@@ -104,8 +104,13 @@ const badgeScansListReducer = (state = DEFAULT_STATE, action = {}) => {
         lastPage
       };
     }
-    case RECEIVE_SPONSORS: {
-      return { ...state, allSponsors: payload.response.data };
+    case RECEIVE_SPONSORS_WITH_SCANS: {
```

Comment:
@santipalenque the idea behind this change is able to offload api
not query all the data set
please review this change
https://github.com/OpenStackweb/openstack-uicore-foundation/commit/32bf9ddbd837d963fba61b38c53fc0a083e17980
we need to update core ui to allow extra filters at querySponsors
and use[ ](https://github.com/OpenStackweb/openstack-uicore-foundation/blob/144c1b27a9ff05124d121a03d8ba9e9edf6cf22b/src/components/inputs/sponsor-input.js#L18)